traffic_ops/testing/api_contract/v4/data/response_template.json

```diff
@@ -1757,5 +1757,95 @@
                 "type": "string"
             }
         }
+    },
+    "cdn_federation": {
+        "type": "object",
+        "required": [
+            "id",
+            "cname",
+            "ttl",
+            "description",
+            "lastUpdated"
+        ],
+        "properties": {
+            "id": {
+                "type": "integer"
+            },
+            "cname": {
+                "type": "string"
+            },
+            "ttl": {
+                "type": "integer"
+            },
+            "description": {
+                "type": "string"
+            },
+            "lastUpdated": {
+                "type": "string"
+            }
```

Comment:
are we missing some fields here? 
```
    {
        "id": 1,
        "cname": "test.quest.",
        "ttl": 48,
        "description": "A test federation",
        "lastUpdated": "2018-12-05 00:05:16+00",
        "deliveryService": {
            "id": 1,
            "xmlId": "demo1"
        }
    }
```

Reply:
Yes, I missed deliveryService , Added it.